test/helpers/estimateGas.js

```diff
@@ -35,7 +35,7 @@ const estimateBaseGas = async (safe, to, value, data, operation,
   return baseGasEstimate + 32000; // Add aditional gas costs (e.g. base tx costs, transfer costs)
 };
 
-const parseRevert = message => new BigNumber(message.substring(67), 16);
+const parseRevert = message => new BigNumber(message.substring(96));
```

Comment:
Why this number?

Reply:
Is a substring of for example this?: `0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000081bd`

Reply:
it's because the revert message has changed and now the important number comes in a different position.